lib/response.js

```diff
@@ -2,10 +2,10 @@
 
 const Buffer = require('safe-buffer').Buffer
 const http = require('http')
-const Stream = require('stream')
+const stream = require('readable-stream')
```

Comment:
Why this change?

Reply:
Compatibility down to node v4.
Socket.destroy works natively only from node v8.

Reply:
As always, thanks!